test/unit/specs/hive/sorted-set.js

```diff
@@ -1078,17 +1071,10 @@ describe('Hive - Sorted Set Store', function() {
       })
 
       it('fails when values is invalid', async () => {
-        const errorMsg = 'Value must be provided and must be a string.'
+        const errorMsg = 'Value must be provided.'
 
         await expect(() => store.getRank(undefined)).to.throw(errorMsg)
         await expect(() => store.getRank(null)).to.throw(errorMsg)
-        await expect(() => store.getRank(0)).to.throw(errorMsg)
```

Comment:
need to add tests for all those methods to check they are able to accept any JSON value

Reply:
\+